jekyll/_includes/snippets/pipeline-values.adoc

```diff
@@ -6,45 +6,65 @@
+| `pipeline.trigger_parameters.github_app.commit_author_name` | GitHub App | | See GitHub documentation for link:https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads[webhook events and payloads].
 | `pipeline.trigger_parameters.gitlab.commit_author_email` | GitLab | | See Gitlab documentation for link:https://docs.gitlab.com/ee/user/project/integrations/webhooks.html[webhooks] and link:https://docs.gitlab.com/ee/user/project/integrations/webhook_events.html[webhook events].
+| `pipeline.trigger_parameters.github_app.commit_author_email` | GitHub App | | See GitHub documentation for link:https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads[webhook events and payloads].
 | `pipeline.trigger_parameters.gitlab.total_commits_count` | GitLab | | See Gitlab documentation for link:https://docs.gitlab.com/ee/user/project/integrations/webhooks.html[webhooks] and link:https://docs.gitlab.com/ee/user/project/integrations/webhook_events.html[webhook events].
+| `pipeline.trigger_parameters.github_app.total_commits_count` | GitHub App | | See GitHub documentation for link:https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads[webhook events and payloads].
 | `pipeline.trigger_parameters.gitlab.branch` | GitLab | | See Gitlab documentation for link:https://docs.gitlab.com/ee/user/project/integrations/webhooks.html[webhooks] and link:https://docs.gitlab.com/ee/user/project/integrations/webhook_events.html[webhook events].
+| `pipeline.trigger_parameters.github_app.branch` | GitHub App | | See GitHub documentation for link:https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads[webhook events and payloads].
 | `pipeline.trigger_parameters.gitlab.default_branch` | GitLab | | See Gitlab documentation for link:https://docs.gitlab.com/ee/user/project/integrations/webhooks.html[webhooks] and link:https://docs.gitlab.com/ee/user/project/integrations/webhook_events.html[webhook events].
+| `pipeline.trigger_parameters.github_app.default_branch` | GitHub App | | See GitHub documentation for link:https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads[webhook events and payloads].
 | `pipeline.trigger_parameters.gitlab.x_gitlab_event_id` | GitLab | | See Gitlab documentation for link:https://docs.gitlab.com/ee/user/project/integrations/webhooks.html[webhooks] and link:https://docs.gitlab.com/ee/user/project/integrations/webhook_events.html[webhook events].
 | `pipeline.trigger_parameters.gitlab.is_fork_merge_request` | GitLab | | See Gitlab documentation for link:https://docs.gitlab.com/ee/user/project/integrations/webhooks.html[webhooks] and link:https://docs.gitlab.com/ee/user/project/integrations/webhook_events.html[webhook events].
```

Comment:
I need some help to work out if there are GitHub App equivalents here

Reply:
Not currently, we'll have to look into adding this for now!